tasks/tenant/update-manager-image-in-git/tests/test-update-manager.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-update-manager
+spec:
+  description: |
+    Run the update-manager-image-in-git task. It doesn't use a workspace, so there isn't a way to
```

Comment:
Oh, that's interesting. I wouldn't even realize this - it's almost automatic that most of our tasks have the data workspace. This also means that if we make the workspace read-only in some of the tasks (to improve the concurrency slowdowns), we won't be able to store mock calls in that workspace (at least that's what I would expect - that the volume is mounted as RO).

Reply:
Yeah, I mean we can add a workspace just for the tests, but I don't like adding things to the code solely for testing purposes

Reply:
Agreed.